### PR TITLE
updated compat bounds

### PR DESCRIPTION
**Pull request by ExpandingMan**
Ok, somebody already beat me to it and updated DataStructures, but I believe the compat bounds were still too specific, so I still think this should be merged.

A 1.3.4 tag would also be appreciated.

### CODE REVIEW

**Comment by ExpandingMan** [comment]
I also added the [CompatHelper](https://github.com/JuliaRegistries/CompatHelper.jl) YAML.  I'm not 100% sure it's set up correctly, but it can always be fixed later.

**Comment by sbromberger** [comment]
We need to get CI working on this branch.

**Comment by ExpandingMan** [comment]
I'm not entirely sure how to do that, but I'll look into it.

For now, I have changed travis and appveyor to run on julia `1` rather than `1.3`.  This should always give the latest `1` and should not be breaking.

**Comment by sbromberger** [comment]
So, what I think we need to do is copy the `.github` directory from `master` into `stable`, and then change `.github/workflows/ci.yml` to reflect the stable branch. Is that something you'd be comfortable doing? Make it a separate PR to `stable` and I'll merge.

**Comment by ExpandingMan** [comment]
Ok, so travis is dead right?  I can do that in this PR, though I'm not sure if it'll run.

**Comment by ExpandingMan** [comment]
Ah, it looks like it is running.  Ok, let's see how this goes.

**Comment by sbromberger** [comment]
Are you sure you've got the right branch in `ci.yml`? I'm seeing `master`. Shouldn't it be `stable`?

**Comment by ExpandingMan** [comment]
Oops.  Would have helped if I actually looked at what those YAML's said.

**Comment by ExpandingMan** [comment]
Yay, looks like we are all set.  A tag would be greatly appreciated.  I have already bumped the Project.toml package version.

**Comment by sbromberger** [comment]
Gotta figure out how to tag this.

File: Project.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ test = ["Base64", "DelimitedFiles", "Test"]
 
 [compat]
 julia = "1"
-ArnoldiMethod = "0.0.4"
-DataStructures = "0.17.9, 0.18"
-Inflate = "0.1.1"
-SimpleTraits = "0.9.1"
+ArnoldiMethod = "0.0"
```
Comment (jaakkor2) [inline Project.toml:30]: ArnoldiMethod v0.0.5 might have breaking changes from v0.0.4 if I interpret https://julialang.github.io/Pkg.jl/v1/compatibility/ correctly

File: Project.toml [hunk 2]
```diff
@@ -27,7 +27,7 @@ test = ["Base64", "DelimitedFiles", "Test"]
...
+DataStructures = "0.17, 0.18"
+Inflate = "0.1"
+SimpleTraits = "0.9"
```
Comment (jaakkor2) [inline Project.toml:33]: Are SimpleTraits 0.9.0, Inflate 0.1.0, DataStructures 0.17.0-0.17.8 compatible?

Comment (sbromberger) [reply Project.toml:33]: It's passing CI so I think we're ok.

**Comment by sbromberger** [comment]
I am in transit and my devbox is offline so I can't tag a new release right now. Probably in a few weeks.

File: Project.toml (hunk 1 above)
Comment (ExpandingMan) [reply Project.toml:30]: I did not realize that there were again special rules for the last digit...  @sbromberger if you are worried about this, you'll have to change it back.

**Comment by ExpandingMan** [comment]
Ok, the good thing about Pkg is that it makes it easy to check out a branch.  If I notice it in my manifest in a few weeks I'll remind you again.

File: Project.toml (hunk 2 above)
Comment (ExpandingMan) [reply Project.toml:33]: Also these upper bounds are not a guarantee that these packages can't hold *each other* back.  So, for example, if `Inflate` was stuck on DataStructures 0.17, LightGraphs would only ever install 0.17 despite what it says here (fortunately that's not the case anyway).

**Comment by jaakkor2** [comment]
Sorry to bug once more, but is it that nobody in the JuliaGraphs organization https://github.com/orgs/JuliaGraphs/people could help with tagging? 

https://github.com/JuliaGraphs/LightGraphs.jl/pull/1473 was merged already three months ago without seeing a tag.

**Comment by ExpandingMan** [comment]
This is a fairly prominent package in the ecosystem, so if there is a reasonable candidate for granting admin rights around, I think that would be a good idea.

**Comment by sbromberger** [comment]
We have sufficient administrative presence. The issue is that you're asking for a fairly significant action in the middle of a pandemic, at the start of a holiday season, during which one of the core committers is in transit cross-country.

Is there a reason you need this tagged so urgently? Why can't this wait a couple more weeks?

**Comment by sbromberger** [comment]
https://github.com/JuliaRegistries/General/pull/26316

**Pull request merged**